tests/virt_autotest/libvirt_extend_storage_lvm.pm

```diff
@@ -33,10 +33,10 @@ our $lvm_vg_name = 'lvm_vg';
 our $lvm_pool_name = 'guest_image_lvm';
 sub run_test {
     my ($self) = @_;
-
+    my @guests = keys %virt_autotest::common::guests;
     record_info "Prepare Guest Systems";
-    foreach (keys %virt_autotest::common::guests) {
-        start_guests() unless is_guest_online($_);
+    foreach (@guests) {
+        assert_script_run("virsh start $_") unless is_guest_online($_);
```

Comment:
There should be updated `start_guests()` to follow this pr purpose instead of trigger `assert_script_run("virsh start $_")` directly.  

please refer to the latest validation run - https://openqa.suse.de/tests/9756859#step/libvirt_extend_storage_lvm/32, for more details.

Reply:
The hotplugging_cleanup failed because the guest install failed, it is stuck. https://openqa.suse.de/tests/9757052#

Regarding libvirt_extend_storage_lvm failure, the host is being reinstalled. I could not check.